libraries/src/Event/Model/BeforeDeleteEvent.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * Joomla! Content Management System
+ *
+ * @copyright  (C) 2023 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license    GNU General Public License version 2 or later; see LICENSE
```

Comment:
```suggestion
 * @license    GNU General Public License version 2 or later; see LICENSE.txt
```

not doing anymore - it will be quicker for you to search/replace

Reply:
I will update.
But I copied it from some existing file, and I thought it is correct

Reply:
I think @heelc29 did the same in your last pull request ;-)